domains/oxyru.json

```
@@ -4,6 +4,22 @@
         "email": "oxyru.okbro@icloud.com"
     },
     "records": {
-        "CNAME": "oxyrv.github.io"
-    }
+        "CNAME": "oxyrv.github.io",
+         "TXT": ["v=spf1 include:zohomail.eu ~all"],
```

you might want to remove the brackets since its just a single TXT records
```suggestion
         "TXT": "v=spf1 include:zohomail.eu ~all",
```

---

Unnecessary change to be honest, both work eitherway.